tenable_io/api/workbenches.py

```diff
@@ -67,6 +86,97 @@ def asset_vulnerabilities(self, asset_id, date_range=None, filters=None, filter_
                                     path_params={'asset_id': asset_id})
         return VulnerabilityList.from_json(response.text)
 
+    def export_download(self, file_id, stream=True, chunk_size=1024):
+        """Download a file that has been prepared for export.
+
+        :param file_id: The unique identifier of the workbench report being downloaded.
+        :param stream: Default to True. If False, the response content will be immediately downloaded.
+        :param chunk_size: If Stream=False, data is returned as a single chunk.
+            If Stream=True, it's the number of bytes it should read into memory.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: The content iterator for the file.
+        """
+        response = self._client.get('workbenches/export/%(file_id)s/download',
```

Comment:
Should we be checking that the status of the file_id == STATUS_EXPORT_READY before attempting to download?